.circleci/config.yml

```diff
@@ -546,13 +510,27 @@ jobs:
       - conditionally-launch-build-and-store:
           study_key: fon
 
-  app-build-and-store-job:
+  app-build:
```

Comment:
Making job name simpler and follow job naming convention: Name begins with a noun.